gpjax/gps.py

```diff
@@ -70,14 +80,19 @@ def __init__(
         r"""Construct a Gaussian process prior.
 
         Args:
-            kernel: kernel object inheriting from AbstractKernel.
-            mean_function: mean function object inheriting from AbstractMeanFunction.
+        kernel: kernel object inheriting from AbstractKernel.
+        mean_function: mean function object inheriting from AbstractMeanFunction.
```

Comment:
Did you run a linter/formatter on the code? It's odd to see changes like this coming in. The linting workflow will not error as it is not possible to lint docstrings, but I would like the format to be consistent across the codebase, so can you please undo this formatting?